deploy/helm/virtink/values.yaml

```diff
@@ -0,0 +1,77 @@
+# Default values for virtink.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+virtController:
+  name: virt-controller
+  replicas: 1
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  podAnnotations: {}
+  podSecurityContext: {}
+    # fsGroup: 2000
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+  image:
+    repository: smartxworks/virt-controller
+
+virtDaemon:
+  name: virt-daemon
```

Comment:
maybe just hardcode to virt-daemon in template is enouth and be compatible with generate ClusterRole name.